eras/shelley/impl/src/Cardano/Ledger/Shelley.hs

```diff
@@ -13,6 +13,7 @@ import Cardano.Ledger.Crypto (StandardCrypto)
 import Cardano.Ledger.Shelley.Era (ShelleyEra)
 import Cardano.Ledger.Shelley.PParams ()
 import Cardano.Ledger.Shelley.Rules ()
+import Cardano.Ledger.Shelley.Translation ()
```

Comment:
what was this needed for? worth leaving a note?

Reply:
It was a mistake, that's why there is whole separate commit for it: https://github.com/input-output-hk/cardano-ledger/pull/3279/commits/b86e70243cb760d47da981db7fb0251080af8c16

Reply:
Not worth leaving notes, because this is something we have to do in all `Cardano.Ledger.[Era]` module